test/integration/test_app/init.lua

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env tarantool
+
+local fiber = require('fiber')
+fiber.create(function()
+    fiber.sleep(1)
+end)
+
+assert(os.getenv('TARANTOOL_APP_NAME'))
+assert(os.getenv('TARANTOOL_INSTANCE_NAME'))
+assert(os.getenv('TARANTOOL_CFG'))
+assert(os.getenv('TARANTOOL_PID_FILE'))
+assert(os.getenv('TARANTOOL_CONSOLE_SOCK'))
+
+fiber.sleep(0.01) -- let `cartridge start` write pid_file and start listening socket
+-- Copied from cartridge.cfg to provide support for NOTIFY_SOCKET in old tarantool
```

Comment:
Could you move it to helpers?

Reply:
What exactly do you mean?

Reply:
This piece of code that there is in at least two files.

Reply:
If you are OK, I would like to prettify all Lua tests in a separated PR. There are many things to change, possibly this code will be removed.

Reply:
ok